kratos/includes/kratos_components.h

```diff
@@ -21,7 +20,6 @@
 #include <map>
 
 // External includes
-#include <boost/ref.hpp>
```

Comment:
this is no longer needed with c++11 => https://www.boost.org/doc/libs/1_66_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/odeint_in_detail/using_boost__ref.html

anyway it was only used in commented code